lib/parseSty.js

```diff
@@ -0,0 +1,133 @@
+"use strict";
+
+var balanced = require("balanced-match");
+
+var declarationCommands = ["\\newcommand",
+                           "\\renewcommand",
+                           "\\DeclareRobustCommand",
+                           "\\defSpecFun"];
+
+// Returns a two dimensional array, where each element contains the body of a bracket section and the type of bracket
```

Comment:
Use multiline comments here as in https://github.com/wikimedia/service-template-node/blob/master/routes/ex.js#L44-L49